psl/psl-core/src/common/preview_features.rs

```diff
@@ -8,7 +10,7 @@ macro_rules! features {
     ($( $variant:ident $(,)? ),*) => {
         #[enumflags2::bitflags]
         #[repr(u64)]
-        #[derive(Debug, Copy, Clone, PartialEq, Eq)]
+        #[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
Is the ordering even used for `PreviewFeature` itself? The added `BTreeMap`s use other stuff as keys.